tests/e2e/tidbcluster/serial.go

```diff
@@ -796,7 +796,10 @@ var _ = ginkgo.Describe("[Serial]", func() {
 
 					ginkgo.By("Deploy original TiDB cluster with prev version")
 					utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc, 6*time.Minute, 5*time.Second)
-					selector := MustGetLabelSelectorForComponents(tcName, label.DiscoveryLabelVal) // ingore discovery
+					selector := MustGetLabelSelectorForComponents(tcName,
+						label.DiscoveryLabelVal, // ingore discovery
+						label.TiFlashLabelVal,   // TODO: ingore tiflash because of PR #4358, remove it after prev major version is greater than v1.2.x
+					)
```

Comment:
deploy tiflash v6.0.0 with operator v1.2.7 and upgrade operator to v1.3.x, tiflash will recreate because of #4358